### PR TITLE
docs(plugins): basic update to splitChunks.automaticNameMaxLength docs

### LINKED ISSUE #3133: splitChunks.automaticNameMaxLength to be documented
__Feature to document__
`splitChunks.automaticNameMaxLength` as per https://github.com/webpack/webpack/releases/tag/v4.35.0

__Author(s)__
@sokra @ryandrew14  

__Additional information__

[ ] I am willing to work on this issue and submit a pull request.

### PR DESCRIPTION
**Pull request by ryandrew14**
_describe your changes..._

This is basic documentation for the new `splitChunks.automaticNameMaxLength` feature, PR'd [here](https://github.com/webpack/webpack/pull/9285). The PR closes #3133. Let me know if you think this needs more examples; I haven't added any yet, as the similar `splitChunks.automaticNameMaxLength` has none, and I think it's a pretty self-explanatory feature.

### CODE REVIEW

File: src/content/plugins/split-chunks-plugin.md [hunk 1]
```diff
@@ -86,6 +87,11 @@ module.exports = {
 
 By default webpack will generate names using origin and name of the chunk (e.g. `vendors~main.js`). This option lets you specify the delimiter to use for the generated names.
 
+### `splitChunks.automaticNameMaxLength`
+
+`number`
```
Comment (EugeneHlushko) [inline src/content/plugins/split-chunks-plugin.md]:
```suggestion
`number: 109`
```

File: src/content/plugins/split-chunks-plugin.md [hunk 2]
```diff
@@ -86,6 +87,11 @@ module.exports = {
...
+
+This option lets you specify a maximum name length to which the chunk names generated by the SplitChunksPlugin will be truncated. By default, Webpack's chunk names are truncated at 109 characters, which is on the long side and is done to prevent errors.
```
Comment (EugeneHlushko) [inline src/content/plugins/split-chunks-plugin.md]:
```suggestion
Allows setting a maximum character count for chunk names that are generated by the `SplitChunksPlugin`.
```

File: src/content/plugins/split-chunks-plugin.md [hunk 3]
```diff
@@ -86,6 +87,11 @@ module.exports = {
...
```
Comment (EugeneHlushko) [inline src/content/plugins/split-chunks-plugin.md]:
```suggestion

__webpack.js.org__

```javascript
module.exports = {
  //...
  optimization: {
    splitChunks: {
      automaticNameMaxLength: 18
    }
  }
}
```

```

**Comment by ryandrew14** [comment]
@EugeneHlushko I like the default thing, but I honestly think my edit is more consistent with the docs on this page. Most have a "this x does y" structure, which I tried to follow as well. I can update to remove the discussion of default values

**Comment by EugeneHlushko** [comment]
Thanks!

**Pull request merged**